### PR TITLE
Added two keyboards

### PR DESCRIPTION
**Pull request by andresteare**
Hi! I'm just a keyboard enthusiast that converted two keyboard (one with a dead MCU and a typewriter) to run with a Teensy! The keyboards are:
- Gamdias 7 colors fullsize iso keyboard
- IBM 6715 typewriter with (buckling springs!)

Note on the IBM one: both shift keys share the same circuit so, to avoid any confussion, I excluded RShift from the keymap (I put bigger warning in the README)

Both of them work flawlessly, I started the idea of supporting the community because I gave the Gamdias to a friend who doesn't even know what QMK is so having the keyboard officially supported by the online configurator would be very helpful. But then decided to pull all the keyboard, because why not help the community :)

I'm a noob when it comes to github, so I hope my request meets the expected standars :)
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes. (I'm not sure what you mean by cover but I test and compile after every change I make)
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/ibm_wheelwriter/config.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#pragma once
```
Comment (drashna) [inline keyboards/handwired/ibm_wheelwriter/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/ibm_wheelwriter/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/handwired/ibm_wheelwriter/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/ibm_wheelwriter/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/handwired/ibm_wheelwriter/keymaps/via/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/ibm_wheelwriter/config.h (hunk 1 above)
Comment (andresteare) [reply keyboards/handwired/ibm_wheelwriter/config.h:17]: done, hope a did it right :)

File: keyboards/handwired/ibm_wheelwriter/keymaps/default/keymap.c (hunk 2 above)
Comment (andresteare) [reply keyboards/handwired/ibm_wheelwriter/keymaps/default/keymap.c:17]: done :)

File: keyboards/handwired/ibm_wheelwriter/keymaps/via/keymap.c (hunk 3 above)
Comment (andresteare) [reply keyboards/handwired/ibm_wheelwriter/keymaps/via/keymap.c:17]: done :)

File: keyboards/.vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (drashna) [inline keyboards/.vscode/settings.json:1]: Please remove this file.

File: keyboards/handwired/ibm_wheelwriter/config.h (hunk 1 above)
Comment (drashna) [reply keyboards/handwired/ibm_wheelwriter/config.h:17]: Yup! and thanks!

File: keyboards/.vscode/settings.json (hunk 4 above)
Comment (andresteare) [reply keyboards/.vscode/settings.json]: done, I don't know if it was necessary but I removed the file AND the folder after comparing to other contributor's repository and noted no one had that foldet in /keyboards, probably I moved the folder by accident

Comment (andresteare) [reply keyboards/.vscode/settings.json]: > Please remove this file.
This is strange, Travis-ci is failing for errors in other contributor's keyboards that are not mine, help

Comment (drashna) [reply keyboards/.vscode/settings.json]: That's fine.  Sometimes Travis CI ... doesn't know what it's doing.   As long as your keyboard compiles, that's what we're looking for.

**Review by drashna — APPROVED** [review]

File: keyboards/.vscode/settings.json (hunk 4 above)
Comment (fauxpark) [reply keyboards/.vscode/settings.json]: You may need to rebase this PR onto current master for Travis to see the correct changes and build accordingly.

Comment (andresteare) [reply keyboards/.vscode/settings.json]: > You may need to rebase this PR onto current master for Travis to see the correct changes and build accordingly.

Please tell that doesn't mean I have to go through the entire PR process again :(

Comment (fauxpark) [reply keyboards/.vscode/settings.json]: Nope, you can just rebase and force push to the same branch and the PR will be updated.

Comment (andresteare) [reply keyboards/.vscode/settings.json]: I don't understand what you mean by "rebase"

Comment (drashna) [reply keyboards/.vscode/settings.json]: This: https://docs.qmk.fm/#/newbs_git_resolving_merge_conflicts?id=rebasing-your-changes

Comment (andresteare) [reply keyboards/.vscode/settings.json]: That fixed it! thanks! :)

File: keyboards/handwired/ibm_wheelwriter/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        KC_TRNS,                                KC_TRNS,                            KC_TRNS,                                                       KC_TRNS,      KC_TRNS
+    ),
+    [3] = LAYOUT(        
+        KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS,
+        KC_TRNS,       KC_TRNS,      KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS,      KC_TRNS,
+        KC_TRNS,       KC_TRNS,        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,               KC_TRNS,
+        KC_TRNS,       KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,
+        KC_TRNS,                                KC_TRNS,                            KC_TRNS,                                                       KC_TRNS,      KC_TRNS
+    ),
```
Comment (fauxpark) [inline keyboards/handwired/ibm_wheelwriter/keymaps/default/keymap.c:47]:
```suggestion
```

File: keyboards/handwired/ibm_wheelwriter/rules.mk [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+# generated by KBFirmware JSON to QMK Parser
+# https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (fauxpark) [inline keyboards/handwired/ibm_wheelwriter/rules.mk:25]:
```suggestion
```

File: keyboards/handwired/oem_iso_fullsize/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS,      KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,       KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                                            KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS,   KC_TRNS,   KC_TRNS,                      KC_TRNS,                              KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,      KC_TRNS, KC_TRNS, KC_TRNS,   KC_TRNS,          KC_TRNS
+    ),
+
```
Comment (fauxpark) [inline keyboards/handwired/oem_iso_fullsize/keymaps/default/keymap.c:52]:
```suggestion
```

File: keyboards/handwired/oem_iso_fullsize/rules.mk [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+# generated by KBFirmware JSON to QMK Parser
+# https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (fauxpark) [inline keyboards/handwired/oem_iso_fullsize/rules.mk:25]:
```suggestion
```

File: keyboards/handwired/ibm122m/keymaps/andresteare/keymap.c [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+    KC_TRNS,    KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,    KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,                       KC_TRNS,             KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,    KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,  KC_TRNS,      KC_TRNS,  KC_TRNS,              KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS,    KC_TRNS,    KC_TRNS,  KC_TRNS,                                    KC_TRNS,                                                            KC_TRNS,  KC_TRNS,                       KC_TRNS,             KC_TRNS,          KC_TRNS
+  ),
+
```
Comment (fauxpark) [inline keyboards/handwired/ibm122m/keymaps/andresteare/keymap.c:62]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/ibm_wheelwriter/readme.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
+# ibm_wheelwriter
+
+Top View of the Keyboard: (https://imgur.com/gallery/4AVq783)
+
```
Comment (fauxpark) [inline keyboards/handwired/ibm_wheelwriter/readme.md:4]:
```suggestion
![top view](https://i.imgur.com/jwYI2qxl.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/ibm_wheelwriter/readme.md (hunk 10 above)
Comment (andresteare) [reply keyboards/handwired/ibm_wheelwriter/readme.md]: thank you!

**Review by fauxpark — APPROVED** [review]

**Comment by andresteare** [comment]
So what know? it looks like there's no QMK collaborator review request

**Comment by drashna** [comment]
Just for somebody to merge it. :)

**Comment by andresteare** [comment]
> Just for somebody to merge it. :)

Thank you! :)

**Pull request merged**